core/runtime/op_driver/futures_unordered_driver.rs

```diff
@@ -241,6 +229,18 @@ impl<C: OpMappingContext> OpDriver<C> for FuturesUnorderedDriver<C> {
   fn len(&self) -> usize {
     self.len.get()
   }
+
+  fn stats(&self) -> OpInflightStats {
+    let q = self.queue.queue.queue.borrow();
```

Comment:
`queue.queue.queue.queue.queue.queue.queue.queue.queue.queue.` 😅

Reply:
It's starting to get a little nested 😓